imports/ui/components/music/playlists.html

```diff
@@ -0,0 +1,17 @@
+<template name="musicPlaylists">
+  <h1>Playlists</h1>
+  <div>
+    {{#if ready}}
+    <table class="playlist-list" border="1">
+    {{#each docs}}
+      <tr>
+        <td><img height="150px" src="{{img showId}}" /></td>
+        <td>{{showName showId}}</td>
+        <td><a href="/playlist/{{spinPlaylistId}}/">{{showDate}}</a></td>
+      </tr>
+    {{/each}}
+    </table>
+    {{> defaultBootstrapPaginator pagination=tempPag limit=10}}
+    {{/if}}
+  </div>
+</template>
```

Comment:
Really excited to check out these new templates. I'm gonna crash right after the code review but I'll look first thing in the morning. It will be a nice break from this Accounts bug, which has been driving me crayyyyzeeee